Thanks — the three synced properties were right and are now all in: `slf4j.version` 2.0.18, `maven.resolver.version` 2.0.21, `jline.version` 4.3.1, plus the `jline-terminal-jansi` → `jansi-core` rename in `dependencyManagement` and the removal of the dead `<jansi.version>` (nothing in the tree references it). Confirmed against the rc-6 distribution itself: `jansi-core-4.3.1.jar` is present, there is no `jline-terminal-jansi` jar, and resolver/slf4j are at 2.0.21/2.0.18.

On #1696 superseding this one — I do not think it does, on three counts:

- it sets `maven.version` to `4.0.0-SNAPSHOT` (plus `apache-snapshots` `<repositories>`/`<pluginRepositories>` blocks), not the released `4.0.0-rc-6`. Its title says "upcoming rc-6" — it was written on 27 July, before rc-6 existed.
- it does not touch `.mvn/wrapper/maven-wrapper.properties`, so the repo would still build *itself* with rc-5 while targeting rc-6.
- it is a draft, `mergeable: false` / `rebaseable: false` against a base from a week ago (it diffs from jline 3.30.13 and resolver 2.0.18, whereas master is now on 3.30.16 and 2.0.20), and it has no CI runs at all.

Happy to be overruled if the intent is to track `4.0.0-SNAPSHOT` rather than pin the release — but in that case the wrapper still needs updating separately.

One note on the remaining red: the GraalVM `Build native distribution` jobs fail on master too (run 30862194274 on 67d0a23d, `undefined reference to stat@GLIBC_2.17` on ubuntu-24.04-arm), so that failure is not from this PR. #1712 is the fix for that one.

--- a/.mvn/release-settings.xml
+++ b/.mvn/release-settings.xml
@@ -1,24 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <settings xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.1.0 http://maven.apache.org/xsd/settings-1.1.0.xsd" xmlns="http://maven.apache.org/SETTINGS/1.1.0"
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-    <profiles>
-        <profile>
-            <id>maven-staging</id>
-            <repositories>
-                <repository>
-                    <id>maven-staging-rc1</id>
-                    <url>https://repository.apache.org/content/repositories/maven-2247/</url>
-                    <releases>
-                        <enabled>true</enabled>
-                    </releases>
-                    <snapshots>
-                        <enabled>false</enabled>
-                    </snapshots>
-                </repository>
-            </repositories>
-        </profile>
-    </profiles>
-    <activeProfiles>
-        <activeProfile>maven-staging</activeProfile>
-    </activeProfiles>
 </settings>

